flow-ide.py

```diff
@@ -268,71 +341,107 @@ def on_selection_modified_async(self, view):
             '// @flow' not in deps.contents and
             '/* @flow */' not in deps.contents
         ):
-            return view.erase_regions('flow_error')
+            view.erase_regions('flow_error')
+            view.erase_regions('flow_uncovered')
+            return
 
         scope = view.scope_name(deps.cursor_pos)
         if 'source.js' not in scope:
             return
 
-        flow = find_flow_bin(deps.project_root)
+        flow = find_flow_bin(deps.project_root, view.window().project_data())
 
+        # START OF FLOW ERROR DISPLAY
         result = call_flow_cli(deps.contents, [
             flow, 'check-contents',
             '--from', 'nuclide',
             '--json',
             deps.filename
-        ])
+        ], view)
 
         if result:
             if result['passed']:
                 view.erase_regions('flow_error')
                 view.set_status('flow_error', 'Flow: no errors')
-                return
+            else:
```

Comment:
Can we do an early `return` before this so that all the coverage code isn't indented so far?